framework/files/usr/lib/systemd/system/elemental-register.service

```diff
@@ -3,10 +3,19 @@ Description=Elemental Register
 Documentation=https://elemental.docs.rancher.com
 Wants=network-online.target
 After=network-online.target
+# Backoff after 5 attempts in 5 min 
+StartLimitIntervalSec=5min
```

Comment:
Out of curiosity. Do we know if once the limit is achieved the timer will restart the counter?